### PR TITLE
control mobs with turned off AI

### PR DESCRIPTION
**Pull request by noskill**

### CODE REVIEW

File: src/main/java/io/singularitynet/Client/ClientStateMachine.java [hunk 1]
```diff
@@ -205,10 +215,40 @@ public ClientStateMachine(ClientState initialState, IMalmoModClient malmoModClie
...
+                    this.controllableEntities.put(uuid, (MobEntity)entity);
+                }
+                removeUuid(uuid);
+            } finally {
+                mobLock.unlock();
+            }
+        }
+    }
+
+    private void onEntityUnoad(Entity entity, ClientWorld clientWorld) {
```
Comment (DaddyWesker) [inline src/main/java/io/singularitynet/Client/ClientStateMachine.java]: Shouldn't this method be named onEntityUn**l**oad?

File: src/main/java/io/singularitynet/Server/ServerStateMachine.java [hunk 2]
```diff
@@ -102,8 +100,35 @@ public ServerStateMachine(ServerState initialState, MissionInit minit, Minecraft
         ServerLifecycleEvents.SERVER_STOPPED.register(this::onServerStopped);
         ServerLifecycleEvents.SERVER_STARTED.register(this::onServerStarted);
         ServerEntityEventsVereya.BEFORE_ENTITY_ADD.register(this::onGetPotentialSpawns);
+        ServerEntityEvents.ENTITY_UNLOAD.register(this::onEntityUnoad);
+        ServerEntityEvents.ENTITY_LOAD.register(this::onEntityLoad);
     }
 
+    private void onEntityUnoad(Entity entity, ServerWorld serverWorld) {
```
Comment (DaddyWesker) [inline src/main/java/io/singularitynet/Server/ServerStateMachine.java]: same here. onEntityUn**l**oad, not Unoad. Or?

**Comment by DaddyWesker** [comment]
Okay, I can approve it, but are we leaving OnEntity**Unoad** ?

**Comment by DaddyWesker** [comment]
All is good now. Though i can't approve for some reason.

**Review by Necr0x0Der — APPROVED** [review]

**Pull request merged**